app/controllers/assignments_controller.rb

```diff
@@ -15,6 +15,8 @@
 along with SSID.  If not, see <http://www.gnu.org/licenses/>.
 =end
 
+require 'zip'
```

Comment:
It wasn't required before?

Reply:
Yup, it was not required before. But it is not a new library that we are using in the application. Here, we are using the rubyzip library which was already used in another controller to extract the files from the zip file. We are using it here to verify whether an uploaded file is zip by using the rubyzip library

Reply:
Ok, thanks for the clarification.